pkg/apis/serving/v1alpha1/revision_types.go

```diff
@@ -118,6 +118,16 @@ const (
 	RevisionContainerConcurrencyMax RevisionContainerConcurrencyType = 1000
 )
 
+const (
+	// RevisionContainerUserPortName is the name that will be used for the Port on the
+	// Deployment and Pod created by a Revision. This name will be set regardless of if
+	// a user specifies a port or the default value is chosen.
+	RevisionContainerUserPortName = "user-port"
```

Comment:
If we were to add named port support, this should probably just be a default instead of a constant?

Reply:
This is the name that gets placed on the Deployment object. To preserve existing behavior I was going to have the Revision controller consume the name rather than passing it through to the Deployment and Pod. I could be convinced otherwise if you believe there is value in having content-negotiation name plumbed through.

Also cant figure out how to comment on the overall comment above, but that change was removed. The list in the commit message is from: https://github.com/knative/serving/pull/2190/commits as that is what this change is rebased, edited, and squashed from.

Reply:
Oh, my mistake! Yeah, the deployment port name has no impact on #2539.